rero_ils/modules/loans/api.py

```diff
@@ -569,7 +569,10 @@ def patron_profile(patron):
                     patron.patron['barcode'])
             requests.append(loan)
         elif loan['state'] in [
-                LoanState.ITEM_RETURNED, LoanState.CANCELLED]:
+                LoanState.ITEM_RETURNED,
```

Comment:
Mention this fix in the PR commit message, specially there is an issue that can be closed with this fix.